nodejs/week1/avg.js

```diff
@@ -0,0 +1,12 @@
+// Excluding "node" and script name
+const args = process.argv.slice(2);
+
+// Convert to numbers
+const numbers = args.map(Number).filter(n => !isNaN(n));
```

Comment:
I see you decided to ignore arguments that are not numbers. Nice implementation! ⚡